cmd/mt-kafka-mdm-report-out-of-order/applicationflags.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+type ApplicationFlags struct {
+	flagSet *flag.FlagSet
+
+	runDurationStr   string
+	ConfFile         string
+	PartitionFrom    int
+	PartitionTo      int
```

Comment:
This is pretty janky because kafkamdm reads this from the config file while the cassandra index doesn't. You want them to be in sync but I don't see a way to extract this from kafkamdm. Maybe there's a way to register this configuration setting here instead of having the use provide it?

This was pulled from `mt-index-prune`. Maybe accepting a string/list at the very least would be closer to how it's specified in the config.